habitat-lab/habitat/datasets/pointnav/pointnav_dataset.py

```diff
@@ -90,15 +90,23 @@ def _get_scenes_from_folder(
         scenes.sort()
         return scenes
 
+    def _load_from_file(self, fname: str, scenes_dir: str) -> None:
```

Comment:
This refactor doesn't seem important now, but I am preparing to support new dataset formats.